index.html

```diff
@@ -1,47 +1,45 @@
 <!DOCTYPE html>
 <html lang="en-us">
-
+​
 <head>
     <meta charset="UTF-8" />
     <link rel="stylesheet" href="./assets/css/style.css">
-    <title>website</title>
+    <title>Horiseon</title>
 </head>
-
+​
 <body>
     <div class="header">
         <h1>Hori<span class="seo">seo</span>n</h1>
         <div>
+        <nav>
             <ul>
-                <li>
-                    <a href="#search-engine-optimization">Search Engine Optimization</a>
-                </li>
-                <li>
-                    <a href="#online-reputation-management">Online Reputation Management</a>
-                </li>
-                <li>
-                    <a href="#social-media-marketing">Social Media Marketing</a>
+                <li><a href="#search-engine-optimization">Search Engine Optimization</a> </li>
+                <li><a href="#online-reputation-management">Online Reputation Management</a></li>
+                <li><a href="#social-media-marketing">Social Media Marketing</a>
                 </li>
             </ul>
+        </nav>
         </div>
     </div>
+
     <div class="hero"></div>
     <div class="content">
         <div class="search-engine-optimization">
```

Comment:
It seems like you have 3 distinct pieces here as 3 `divs` which can again be replaced by a semantic HTML element. Research in to what element can be used here and replace the `div`